Src/FluentAssertions/Types/TypeAssertions.cs

```diff
@@ -1102,5 +1216,14 @@ protected override string Identifier
         {
             get { return "type"; }
         }
+
+        private void AssertThatSubjectIsClass()
+        {
+            var typeInfo = Subject.GetTypeInfo();
+            if (typeInfo.IsInterface || typeInfo.IsValueType || typeof(Delegate).IsAssignableFrom(typeInfo.BaseType))
+            {
+               throw new InvalidOperationException($"{Subject} must be a class.");
```

Comment:
Last thing I can come up with is a missing space to keep four space indentation.